.github/workflows/run-wp-env/action.yml

```diff
@@ -39,16 +39,17 @@ runs:
     - name: Install wp-env
       shell: bash
       run: |
-          npm i -g --no-package-lock --no-save @wordpress/env@5.7.0
+          npm i -g --no-package-lock --no-save @wordpress/env@8.6.1
     - name: Setup WordPress environment
       shell: bash
       run: |
         npx wp-env start
+        npx wp-env run cli chmod -R 777 wp-content
```

Comment:
Why do we need this?

Reply:
Wordress doesn't allow to write into this directory, so we can install plugin. Actually we have a lot of permissions problem if we use github action to create a new env. That is why I switch to npx wp-env start

Reply:
Shouldn't we add it to the `start-local-server` script?